### DISCUSSION TITLE
How can I use tailwind/TabView and make it on vertical tabs

### DISCUSSION THREAD

**Discussion by AbirMasmoudiSaqara** [primefaces/community#1127] [PrimeVue] [answered]
I would like to have the 'tailwind/tabView' in vertical form as in the image.
![image](https://github.com/primefaces/community/assets/93143899/ba39f68b-7e65-4ca5-a56d-86be3ff87376)

**Comment by tugcekucukoglu** [comment, marked as answer]
TabView does not have vertical mode, but if you want, you can make your own TabView component with or without Tailwind.
